### PR TITLE
Add table except

### PR DESCRIPTION
**Pull request by zbrookle**
This pull request adds the SQL 'EXCEPT' functionality, which is the set difference. I followed the same format as intersect so it should be fairly straightforward

### CODE REVIEW

File: ibis/pandas/execution/tests/test_operations.py [hunk 1]
```diff
@@ -839,6 +839,18 @@ def test_intersect(client, df1, intersect_df2):
     tm.assert_frame_equal(result, expected)
 
 
+def test_except(client, df1, intersect_df2):
```
Comment (jreback) [inline ibis/pandas/execution/tests/test_operations.py:842]: let's call this difference as its a much more common name than except

Comment (zbrookle) [reply ibis/pandas/execution/tests/test_operations.py:842]: Ok sounds good, I'll update the names for everything

File: ibis/sql/compiler.py [hunk 2]
```diff
@@ -1011,6 +1017,11 @@ def _get_keyword_list(self):
         return ["INTERSECT" for _ in range(len(self.tables) - 1)]
 
 
+class Difference(SetOp):
+    def _get_keyword_list(self):
+        return ["EXCEPT" for _ in range(len(self.tables) - 1)]
```
Comment (datapythonista) [inline ibis/sql/compiler.py:1022]: This should be the same, but clearer, right?
```suggestion
        return ["EXCEPT"] * (len(self.tables) - 1)
```

**Review by datapythonista — APPROVED** [review]
lgtm, thanks @zbrookle

File: ibis/sql/compiler.py (hunk 2 above)
Comment (zbrookle) [reply ibis/sql/compiler.py:1022]: Yes I could change it to that

**Comment by zbrookle** [comment]
@jreback I know this is a feature but would it be possible to have it in the next bug fix release so that it gets shipped alongside the intersection feature from #2250 ?

**Comment by jreback** [comment]
> @jreback I know this is a feature but would it be possible to have it in the next bug fix release so that it gets shipped alongside the intersection feature from #2250 ?

these are just labels am assigning. we havent' been doing bug fix releases anyhow (only major ones), so will be included.

**Comment by zbrookle** [comment]
@jreback Ok sounds good!

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @zbrookle

**Pull request merged**